src/media/css/feed/shelves.styl

```diff
@@ -121,3 +150,32 @@ shelf_curve_color($color) {
 [data-shelf-landing-carrier="mtn"] {
     collection_background_colors($mtn_color);
 }
+
+@media $base-tablet {
+    [data-shelf-landing] {
+        .feed-landing-header-mobile {
+            display: flex;
+
+            h1 {
+                font-size: 30px;
+                font-weight: 100;
+                line-height: 40px;
+                top: -7px;
+            }
+        }
+        .feed-landing-header-desktop {
+            display: none;
+        }
+    }
+}
+
+@media $base-desktop {
+    [data-shelf-landing] {
+        .feed-landing-header-mobile {
+            display: none;
+        }
+        .feed-landing-header-desktop {
+            display: flex;
+        }
```

Comment:
They are pretty different. Plus, when I was implementing it, I wanted to keep mobile the same while integrating new desktop designs. So mobile just reuses what it looks like on Feed, and desktop is something different. We could do some responsive stuff, but it'd be pretty messy to try to stretch something into something completely different. And I don't think we save that much code doing so.